source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -212,21 +214,22 @@ export abstract class BaseSerializationContext {
     }
 
     logEvent(
+        source: SerializableObject | undefined,
         phase: Enums.ValidationPhase,
         event: Enums.ValidationEvent,
-        message: string,
-        source?: SerializableObject) {
+        message: string) {
```

Comment:
something to consider as a future enhancement -- if there's a way we could strongly type the string helpers (e.g. via a type alias or something), it would be an easy way to enforce the "don't use a hardcoded string" rule :)